id.mk

```diff
@@ -1,8 +1,10 @@
 ## Identify drafts, types and versions
 
-ifneq (,$(shell git submodule status $(LIBDIR) 2>/dev/null))
+ifneq (,$(wildcard .gitmodules))
+ifneq (,$(shell grep "path = $(LIBDIR)" .gitmodules))
```

Comment:
You can use `$(and $(wildcard ...),$(shell ... ))` and inline this.  Or you can observe that grep on a non-existent file produces no output to stdout and just do:

ifneq (,$(grep "path *= *$(LIBDIR)" .gitmodules >/dev/null))